values.yaml

```diff
@@ -276,7 +276,7 @@ pingProber:
     pullPolicy: IfNotPresent
 
 pingExporter:
-  enabled: true
```

Comment:
As this will break backward compatibility with all current deployments, this will not be changed until 1.0 as foreseeable now. Please reference, that this is a problem in the documentation.